app/models/state_file_az_intake.rb

```diff
@@ -181,6 +181,10 @@ def requires_hoh_qualifying_person_name?
   end
 
   def hoh_qualifying_person_name
+    unless requires_hoh_qualifying_person_name?
```

Comment:
I think, and I never say this, that this could be a `return unless ...` one-liner